tools/tco/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+#===-- tools/tco/CMakeLists.txt --------------------------------------------===#
```

Comment:
Is there a need for header here? I may be wrong, but I believe CMakeLists.txt does not need the header.

Reply:
Appears the header rewrites got overly exuberant.